### DISCUSSION TITLE
Proposal to change .NET's name to Vulcan

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/csharplang#5650] [Language Ideas] [answered]
I would like .NET's name be changed to something cooler, something that doesn't have the same length of characters as 'cat', something that people can know faster that it's a piece technology that's cool rather than something that just randomly sits there.

If you are like minded, feel free to leave a thumbs up

**Comment by HaloFour** [comment, marked as answer]
Such a suggest really has nothing to do with the C# language.  I'd suggest posting to the [runtime](https://github.com/dotnet/runtime/) repository but I doubt it's their concern either.

I also seriously doubt such a suggestion will get the least bit of traction.  You can't get "cooler" than 20+ years of already established.

**Reply by DavidArno** [reply]
My gut feeling is anyone young enough to worry about whether a language is "cool" wouldn't be seen dead near C# and instead prefers to torture themselves for fun with JavaScript. 😇

**Reply by benbhall** [reply]
I'm not sure that being cool is high up on the priorities of your typical Vulcan :)

**Reply by Xyncgas** [reply]
oof, hey I invent algorithms for AI on dotnet, I've never touched javascript and I don't need to thanks to ASP.NET's (As sooon as possible dot net) Blazor

**Comment by jnm2** [comment]
This would also be hugely confusing given the existence of https://www.vulkan.org/ and the .NET bindings for Vulkan.

**Reply by HaloFour** [reply]
You mean the Vulcan bindings for Vulkan?
